test/cppsuite/configs/operations_test_default.txt

```diff
@@ -65,6 +65,12 @@ workload_generator=
         ops_per_transaction=(max=100,min=50),
         thread_count=10
     ),
+    remove_config=
+    (
+        op_rate=500ms,
+        ops_per_transaction=(max=100,min=0),
```

Comment:
We should make max less, since its more of a default config.

Reply:
Done in b8066f503c28d155e09f187b88472a268d760ca5